src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,44 @@
+.stars--4 {
+  display: flex;
+  padding-top: 16px;
+
+  & .stars__star:nth-child(1) {
+    display: block;
+    background-image: url(../images/star-active.svg);
+    width: 13px;
+    height: 12.2px;
+    margin-left: 17px;
+  }
+
+  & .stars__star:nth-child(2) {
+    display: block;
+    background-image: url(../images/star-active.svg);
+    width: 13px;
+    height: 12.2px;
+    margin-left: 4px;
+  }
+
+  & .stars__star:nth-child(3) {
+    display: block;
+    background-image: url(../images/star-active.svg);
+    width: 13px;
+    height: 12.2px;
+    margin-left: 4px;
+  }
+
+  & .stars__star:nth-child(4) {
+    display: block;
+    background-image: url(../images/star-active.svg);
+    width: 13px;
+    height: 12.2px;
+    margin-left: 4px;
+  }
+
+  & .stars__star:nth-child(5) {
+    display: block;
+    background-image: url(../images/star.svg);
+    width: 13px;
+    height: 12.2px;
+    margin-left: 4px;
```

Comment:
The styles for `.stars__star` are repeated with the exception of the `background-image` for the last star. Consider using a common class to define shared properties and only modify the `background-image` when necessary to follow the DRY (Don't Repeat Yourself) principle. For example, you can create a `.stars__star` class that contains all the shared styles and then add a modifier class such as `.stars__star--inactive` for the different background image.